lib/views/options.html

```diff
@@ -155,6 +155,17 @@
                     <span style="{{opctrl.translatedWordStyle}}">awesome</span>
                     style!
                   </p>
+                  <small>
+                    <span class="label label-danger">Note:</span>
+                    <span style = "color: rgba(75, 174, 79, 1.00);">Green, </span>
+                    <span style = "color: rgba(164, 59, 64, 1.00);">Mexican Red</span>
```

Comment:
Is mexican red really the name of this color? Maybe we should use a color that doesn't refer to a nationality in its name.